test/routs/soldiersTest.js

```diff
@@ -0,0 +1,66 @@
+const mocha = require("mocha");
+const http = require("http");
+const mongoServer = require("../../start").server;
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+const expect = require('chai').expect;
+const MongoClient = require('mongodb').MongoClient;
+
+// Connection URL
+const mongoUrl = 'mongodb://localhost:27017';
+
+// Database Name
+const dbName = 'myproject';
+
+describe("soldiers api", function () {
+  const url = 'http://localhost:3000/soldiers';
+
+  after("soldiers api", function (doneAfter) {
+    mongoServer.close();
+    doneAfter();
+  });
+
+  describe("soldier POST", function () {
+    it("should return a succesfull respone when the soldier has been added to the database", function (doneIt) {
```

Comment:
the norm is done rather than doneIt. This is arbitrary, yes, but since it's what we use in the group you should get used to it